node/test/go/dev-mode/test/fixtures/lambdas/250ms.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
I am thinking it might be best to just test the `thenable`, `callback`, and `250ms` tests. That way we cover the two different ways runtimeDone could be invokes with the first two tests and the `250ms` test will help test logs that come in after a delay 👍

Reply:
`thenable` and `callback` make a difference only for an internal extension. From an external extension point of view there's no real difference (that part is black box)

Actually, what I would introduce is a single lambda test that issues some logs over some time. e.g. runs for 5 seconds and issues 10 logs every half a second. That would well test that the extension actually gathers those logs on the go and e.g. in the next invocation.

Reply:
This sounds like a good to me 👍 I went ahead an implemented almost exactly what you mentioned